projects/qgrid-core/body/body.let.js

```diff
@@ -17,6 +21,19 @@ export class BodyLet {
 				}
 			});
 
+		observe(model.mouseChanged)
+			.subscribe(({ state }) => {
+				const { code, status, target } = state;
+				if (target && code === 'right' && status === 'up') {
+					selectText(target.element);
+					this.selectedNodes.push(target.element);
+				}
+			});
+
+		const manager = new EventManager(this);
```

Comment:
why we need that, won't be automatically removed by browser? if not can we use the same mouseChange event?